### PR TITLE
fix(script): nightly test

### PR DESCRIPTION
**Pull request by ailisp**
https://nightly.neartest.com/run/c1ba86d68438f22d25235ba66e2254439d55e9c4_200406_171929/expensive/expensive_near_sync_state_nodes_sync_state_nodes_multishard/stderr
: fixed in #2401 
https://nightly.neartest.com/run/c1ba86d68438f22d25235ba66e2254439d55e9c4_200406_171929/pytest/pytest_contracts_deploy_call_smart_contract_py/stderr: fixed here
0 stdout and 0 stderr of state_sync2: Possibly fixed using the same trick used the parallel_run_test.py. Also If still no output, added print to ensure it's stdout cached instead of there's really no output

Test Plan
------------
deploy_call_smart_contract passed locally

### CODE REVIEW

File: pytest/tests/sanity/state_sync2.py [hunk 1]
```diff
@@ -4,16 +4,22 @@
...
+import fcntl
 
 sys.path.append('lib')
 
 from cluster import start_cluster
 from utils import LogTracker
 
+fcntl.fcntl(1, fcntl.F_SETFL, 0) # no cache when execute from nightly runner
+
+print('start state sync2')
```
Comment (SkidanovAlex) [inline pytest/tests/sanity/state_sync2.py:16]: remove at least one of the two traces?

File: pytest/tests/sanity/state_sync2.py [hunk 2]
```diff
@@ -4,16 +4,22 @@
...
 
 import sys, time
+import fcntl
 
 sys.path.append('lib')
 
 from cluster import start_cluster
 from utils import LogTracker
 
+fcntl.fcntl(1, fcntl.F_SETFL, 0) # no cache when execute from nightly runner
```
Comment (SkidanovAlex) [inline pytest/tests/sanity/state_sync2.py:14]: What is this line doing? Disabling buffering?

Comment (ailisp) [reply pytest/tests/sanity/state_sync2.py:14]: I don't remember exactly, it worked to disable buffering in subprocess.PIPE kind of stdout/stderr in gitlab-ci parallel-run-test.py (which is also when a pytest launched by nightly.py)

File: pytest/tests/sanity/state_sync2.py (hunk 1 above)
Comment (ailisp) [reply pytest/tests/sanity/state_sync2.py:16]: want to keep both to see do stdout and stderr both works

**Comment by ailisp** [comment]
> why does this only apply to one test?

It's possible state_sync2 really haven't produce any stdout/stderr at timeout. So add trace to see if it's the case

**Comment by evgenykuzyakov** [comment]
@ailisp `{'local': True, 'near_root': '../target/debug/', 'binary_name': 'near'}` has to be changed to `{'local': True, 'near_root': '../target/debug/', 'binary_name': 'neard'}` in `cluster.py`

**Comment by ailisp** [comment]
> @ailisp {'local': True, 'near_root': '../target/debug/', 'binary_name': 'near'} has to be changed to {'local': True, 'near_root': '../target/debug/', 'binary_name': 'neard'} in cluster.py

Thanks for point out. near still works because neard package also has a near binary, but i'll update

**Pull request merged**